GVRf/Sample/gvr-video/src/org/gearvrf/video/VideoScript.java

```diff
@@ -410,6 +410,12 @@ public void run() {
                     mGVRContext, "button/play-active.png"));
             mPlayPauseButton = new GVRSceneObject(gvrContext,
                     gvrContext.createQuad(0.7f, 0.7f), mInactivePause);
+            mPlayPauseButton.getRenderData().getMaterial().setTexture("active_play", mActivePlay);
```

Comment:
It is no less confusing to me than the current sample that sets the texture to key "active_play" in the material, and when actually needed for rendering - sets it as the main texture. What "active_play" is for is completely opaque and pretty much anyone who looks at the code for the first time will not get it. While I am anything but advocate of setKeepWrapper, its use would raise the awareness of how things work and how the framework does its own reference counting.

It may be a kludge but it is a highly visible method. Too late to try hiding it.